DoctorWho.Web/Validation/DoctorManipulationResourceValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using DoctorWho.Web.Resources;
+using FluentValidation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DoctorWho.Web.Validation
+{
+    public class DoctorManipulationResourceValidator : AbstractValidator<DoctorManipulationResource>
+    {
+        public DoctorManipulationResourceValidator()
+        {
+            RuleFor(resource => resource.LastEpisodeDate).Null().When(resource => resource.FirstEpisodeDate == null);
```

Comment:
breaking this to multi-line makes it more readable:
![image](https://user-images.githubusercontent.com/44105316/161428138-10ce3543-6285-46a6-a923-21afde65aac3.png)